modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/impl/RestAPIStoreImpl.java

```diff
@@ -672,24 +672,19 @@ public HttpResponse addApplicationWithGroup(String application, String tier, Str
      * @return - http response of add application
      * @throws APIManagerIntegrationTestException - if fails to add application
      */
```

Comment:
Method comment should be updated with the new return type.

Reply:
fixed in 
151f65e